### PR TITLE
Fix a bunch of issues

### PR DESCRIPTION
**Pull request by Lyfts**
The mysterium transformer wasn't being initiated which meant that its [weird patches](https://github.com/GTNewHorizons/Backhand/blob/master/src/main/java/net/tclproject/mysteriumlib/asm/fixes/MysteriumPatchesFixesO.java) (wow this could use a mixin conversion) were never applied. Instead of having 2 coremods I merged the mysterium fix loader doohickey into the Backhand coremod and added the two missing transformers.
This fixed the majority of the problems and
Fixes https://github.com/GTNewHorizons/Backhand/issues/4, fixes https://github.com/GTNewHorizons/Backhand/issues/8, fixes https://github.com/GTNewHorizons/Backhand/issues/5 and this also fixes the problem mentioned in https://github.com/GTNewHorizons/Backhand/issues/2

I also fixed some incorrect interactions with EFR's totem of undying, tipped arrows & fireworks which
Fixes https://github.com/GTNewHorizons/Backhand/issues/9, fixes https://github.com/GTNewHorizons/Backhand/issues/6, fixes https://github.com/GTNewHorizons/Backhand/issues/7

I also ditched the access transformer class because it was both pointing to the wrong file and not necessary

### CODE REVIEW

File: src/main/java/xonin/backhand/ServerEventsHandler.java [hunk 1]
```diff
@@ -51,24 +52,21 @@ public void onLivingDeath(LivingDeathEvent event) {
...
-                regularHotSwap = true;
-                MinecraftForge.EVENT_BUS.post(event);
-            }
-        } catch (Exception ignored) {}
+        Item totem = ModItems.TOTEM_OF_UNDYING.get();
+
+        if (offhandItem.getItem() == totem && (mainhandItem == null || mainhandItem.getItem() != totem)) {
+            BackhandUtils.swapOffhandItem(player);
+            regularHotSwap = true;
+        }
```
Comment (Caedis) [inline src/main/java/xonin/backhand/ServerEventsHandler.java:69]: Needs a mod guard for EFR

**Review by Caedis — CHANGES_REQUESTED** [review]

File: src/main/java/xonin/backhand/ServerEventsHandler.java (hunk 1 above)
Comment (Lyfts) [reply src/main/java/xonin/backhand/ServerEventsHandler.java]: idfk how I forgot that

**Review by Caedis — APPROVED** [review]

**Comment by huihiuhuai** [comment]
Have you considered forking the 1.4.1 version?

**Comment by Lyfts** [comment]
No

**Comment by Caedis** [comment]
We are actually based on the 1.4.1 version if you check the commit history

**Comment by huihiuhuai** [comment]
> We are actually based on the 1.4.1 version if you check the commit history

but 1.4.1 doesn't have most of the aforementioned issues..

**Pull request merged**